addons/account/models/account_payment_term.py

```diff
@@ -258,7 +258,16 @@ class AccountPaymentTermLine(models.Model):
             ('days_after', 'Days after invoice date'),
             ('days_after_end_of_month', 'Days after end of month'),
             ('days_after_end_of_next_month', 'Days after end of next month'),
+            ('days_end_of_month_on_the', 'Days end of month on the'),
```

Comment:
I myself don't understand what this value means, but that's probably due to the lack of functional knowledge (?). Just to be sure: we do not need to map selection values for this field, correct?

Reply:
It's a totally new options for the selection field of payment term needed for some country (ex: France) so we do not need to map old stuff to this new selection field :smile:

Reply:
Great! I was unsure since it was already defined in the other module, anyways better safe than sorry :p